packages/front-end/components/Tags/TagsFilter.tsx

```diff
@@ -90,9 +71,11 @@ export default function TagsFilter({
           setTags(value);
         }}
         prompt={"Filter by tags..."}
-        autoFocus={open}
```

Comment:
Removed the autoFocus prop since I removed the code above to only show the dropdown after clicking the "Filter by tags..." link.